CHANGES

```diff
@@ -11,14 +30,14 @@ Version 4.1.0 (August 25th, 2021)
 **Bug fixes**
 
 * Update sanitizer clean to use vendored 3.6.14 stdlib urllib.parse to
-  fix test failures on Python 3.9 #536
+  fix test failures on Python 3.9. (#536)
```

Comment:
I went through this file and tweaked some formatting so it was more consistent. The changes are cosmetic after this point.